### PR TITLE
Fixing forums and login translation issues.

### PR DESCRIPTION
**Pull request by ahmedaljazzar**
* Updating translations configuration file for extracting strings from .underscore files located in static.
* Making the translations of login-register template collectible.

### CODE REVIEW

**Comment by OmarIthawi** [comment]
**Bug:** If there's more than one error is is not appearing as a list, instead it appears in a very tight line.

email-pass

File: common/static/js/utils/edx.utils.validate.js [hunk 1]
```diff
@@ -136,15 +144,18 @@ var edx = edx || {};
 
                             // If the field has a custom error msg attached, use it
                             if ( customMsg ) {
-                                tpl = _fn.validate.msg.custom;
+                                // TODO: Escape from XSS
```
Comment (OmarIthawi) [inline common/static/js/utils/edx.utils.validate.js]: Why it is not escaped? Cosider using the form:
`<%- content %>` instead.

**Comment by OmarIthawi** [comment]
@ahmedaljazzar Check the two comments and I think we're good to merge. 

My head is not helping me to do proper merges, but at least I've tested it on my machine and the code looks good.

**Comment by ahmedaljazzar** [comment]
@OmarIthawi Solved.

**Comment by OmarIthawi** [comment]
@ahmedaljazzar Looks good to me. Merging...

**Pull request merged**